initramfs/init.sh

```diff
@@ -183,15 +179,13 @@ set_display_splash()
     echo "Setting display image."
 
     #Get the article number from EEPROM
-    art_num=$(i2ctransfer -y 3 w2@0x57 0x01 0x00 r4)
+    art_num=$(i2ctransfer -y 1 w2@0x57 0x01 0x00 r4)
```

Comment:
Is the reading and writing working for the article number?

Reply:
We were unable to test it because we did not get usb to work

Reply:
We were able to read from the eeprom at this device id/address, but of course it's all 0xFF from factory